SMLHelper/FishFramework/CustomFish.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using SMLHelper.V2.Handlers;
```

Comment:
General cleanup for all files:  
Remove any unused `using` statements.  
Move any remaining  `using` statements inside the namespace.